network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -2,6 +2,23 @@
 
 {% load wagtailcore_tags mini_site_tags %}
 
+{% block coral_talk_meta %}
+  {% if coralTalkServerUrl and page.feature_comments %}
```

Comment:
```suggestion
  {% if coral_talk_server_url and page.feature_comments %}
```

Reply:
Looking at this some more, this check is performed in quite a few places, so it would actually make more sense to set a context variable that captures this AND rather than having an `{% if %}` with these two checks in four separate places in the same template?

Reply:
Sorry, I don't understand what came after the 'AND'.